progression/src/main/resources/progression-migrations/V9__Add_farming_progression_tables.sql

```diff
@@ -0,0 +1,11 @@
+
+create table if not exists progression_farming
+(
+    id varchar(36) not null primary key,
+    Gamer varchar(36) not null,
```

Comment:
Add an index for gamer, and timestamp, and both

CREATE INDEX idx_progression_farming_timestamp ON progression_farming (timestamp);
CREATE INDEX idx_progression_farming_gamer ON progression_farming (Gamer);
CREATE INDEX idx_progression_farming_composite ON progression_farming (timestamp, Gamer);